src/drt/test/Distributed/k8s-drt.yaml

```diff
@@ -35,12 +35,13 @@ spec:
       hostNetwork: true
       containers:
         - name: openroad
-          image: openroad/centos-binary:latest
+          image: openroad/orfs:v3.0-1871-g4e29c449
```

Comment:
Will this image always exist?

Reply:
@vvbandeira @sombraSoft  what do you recommend here for the tag?

Reply:
The image will exist for as long as DockerHub maintains the current policy of allowing unlimited images for open-source projects.

One case would be to use the base ORFS image and compile OR on the fly to guarantee that the code being tested is the current checkout code. The other case might be to use `orfs:latest` and include this in the CI for ORFS and OR to ensure it will always work.

Reply:
@vvbandeira  `orfs:latest` does not appear to be updating as Osama commented.  See https://hub.docker.com/r/openroad/orfs/tags?name=latest

![image](https://github.com/user-attachments/assets/6ffdfbec-1d25-4528-a537-a9ee60da3251)